File: app/src/main/assets/jsexamples/java/ch3/CountSpaces.java
class CountSpaces {
	public static void main(String[] args) throws java.io.IOException {
		char ch;
		int spaces = 0;

		System.out.println("Enter a sentence ending with a period:");
		while ((ch=(char) System.in.read())!='.') 
			if (ch==' ') spaces++;
		System.out.println("There were "+spaces+" spaces entered.");
	}
}

// Enter a sentence ending with a period:
// Hello this is a test.
// There were 4 spaces entered.

File: app/src/main/assets/jsexamples/kotlin/ch11/countSpaces.js
function countSpaces(str) {
	return str.split(' ').length - 1;
}

let s1 = "This is a test."; // 3
console.log(countSpaces(s1));
let s2 = "This is a  test."; // 4
console.log(countSpaces(s2));
let s3 = "  "; 
console.log(countSpaces(s3)); // 2
console.log(s3.split(' ')); // [ '', '', '' ]
